kv/task.go

```diff
@@ -568,7 +569,8 @@ func (s *Service) createTask(ctx context.Context, tx Tx, tc influxdb.TaskCreate)
 		Flux:            tc.Flux,
 		Every:           opt.Every.String(),
 		Cron:            opt.Cron,
-		CreatedAt:       time.Now().UTC().Format(time.RFC3339),
+		CreatedAt:       createdAt,
+		LatestCompleted: createdAt,
```

Comment:
I don't understand this line.  Why are we setting the LatestCompleted to createdAt?

Reply:
When a task is first created we don't have a latest completed time. If a task goes to the scheduler without latest completed it will start at 1970. A while ago we decided to make latest completed never be before createdAt. KV was not following the standard set by the other stores and by updating the test it exposed this difference.